accelerate/nuxtjs-starter/README.md

```diff
@@ -0,0 +1,93 @@
+# Prisma Accelerate Example: Nuxt.js Starter
+
+## Introduction
+The quote generator typically fetches and returns a random quote from a predefined set of quotes. In the context of your Nuxtjs application, the quote generator is a service that provides quotes, potentially with some caching strategy applied to optimize performance and reduce the load on the server.
+
+## General Steps of a Quote Generator:
+- Fetch a Random Quote: It selects a random quote from a collection of quotes.
```

Comment:
As per the label the behaviour should be different.

The label in the landing page states: "Retrieves the most recently added quote with and without caching enabled"

![Uploading image.png…]()

Reply:
```suggestion
- Fetch Quote: Retrieves the most recently added quote with and without caching enabled.
```